### ISSUE TITLE
Looping/Repeating messages and missing messages

### ISSUE THREAD

**Issue by vertigo220** [vector-im/element-web#24354]
### Steps to reproduce

Didn't do anything. I leave Element running all the time, just restored it and started scrolling through a conversation and found that it kept looping the same things over and over. Only part of that is shown in the video, because I didn't want to include other parts of the conversation, but after the multiple loops shown in the video, there were a few to several lines of the chat and then it went back to the same looping messages. What's more, many of the messages between us aren't there, i.e. I keep scrolling up and down from start to end and only a small fraction of the messages we've sent to each other are there. And it gets even stranger, because I'll scroll to the top, then scroll down, past the looping part until I get to some actual lines of conversation, scroll past those, then scroll back up and they'll be gone, and I'll just be right back in the looping, without going back past the lines I just saw. And not only do the lines randomly show, but it's also random _which_ lines show.

Also, not sure if it's related, but my internet connection is a bit flaky, and it went out for a bit while I was doing this, and I had just scrolled all the way from the top of the conversation to the bottom, and the last messages were there, then I scrolled back up and down again, and those messages were no longer showing and I got the error banner at the top about decrypting messages, despite the fact they were _just_ showing a few seconds prior, and the banner remained and the messages didn't show until I got the connection back.

#### What did you expect?
For it to show each message once and only once

#### What happened instead?

See video:

https://user-images.githubusercontent.com/12024557/215239988-bfc9b698-e5f7-4e1e-a83d-11a77d365284.mp4

### Operating system

Windows 10

### Application version

Element version: 1.11.18 Olm version: 3.2.12

### How did you install the app?

From main website

### Homeserver

matrix.org

### Will you send logs?

Yes

---

 - repeat
 - duplicate
 - looping

**Comment by vertigo220** [comment]
logs sent

**Comment by vertigo220** [comment]
Just exited and restarted Element and it's back to normal, so at least all the messages are still there (though they load very slowly as I scroll up, but I assume that's normal, though far from ideal). This is the second issue in the past day or two I've had to restart it to fix, so not sure if it's an issue with my install or if it's really this buggy.

**Comment by richvdh** [comment]
duplicate of #25195
